src/System.Private.CoreLib/shared/System/Globalization/CalendricalCalculationsHelper.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Weren't some of these in your calendars PR, or am I misremembering?

Reply:
I think only this one - will fix that

Reply:
actually a bit more - i need to start merging :(